definitions/mobile.d.ts

```diff
@@ -493,4 +493,32 @@ declare module Mobile {
 	interface IiOSDeviceProductNameMapper {
 		resolveProductName(deviceType: string): string;
 	}
+
+	interface IAndroidDeviceHashService {
+		/**
+		 * Returns the hash file path on device
+		 */
+		hashFileDevicePath: string;
+		/**
+		 * If hash file exists on device, read the hashes from the file and returns them as array
+		 * If hash file doesn't exist on device, returns null
+		 */
+		getShasumsFromDevice(): IFuture<string[]>;
```

Comment:
It is more functunal-style to return an empty array instead of null, but I do not insist.